Gemfile.lock

```diff
@@ -371,7 +378,9 @@ DEPENDENCIES
   fog-google (~> 0.1.0)
   fontello_rails_converter
   grape (~> 1.0)
+  grape-entity (~> 0.7.1)
   grape-swagger (~> 0.28)
+  grape_on_rails_routes (~> 0.3.2)
```

Comment:
is it absolutely necessary can we do it without this rails_routes gem ?

Reply:
@mod, @maksim-litvinov I think it could be moved to `development` section on Gemfile

Reply:
moved